internal/commands/util/utils.go

```diff
@@ -128,3 +135,61 @@ func ReadFileAsString(path string) (string, error) {
 
 	return string(content), nil
 }
+
+func CompressFile(sourceFilePath, targetFileName string) (string, error) {
+	outputFile, err := os.CreateTemp(os.TempDir(), "cx-*.zip")
+	if err != nil {
+		return "", errors.Wrapf(err, "Cannot create temp file")
+	}
+	defer func(outputFile *os.File) {
+		CloseFileErr := outputFile.Close()
```

Comment:
have privte methods for closing the output/zip/data, call from the defer to this mthod